_posts/2023-09-05-gs_00009_vs-code.md

```diff
@@ -0,0 +1,176 @@
+open it for you. So, indexing can be described as generating an address book of
+all variables, functions, types, and so on. **Linting** checks your code while
+you type. It can detect errors like a typo in a variable right away. Don't worry
+if you don't have a picture of these features yet. It will become clear once you
+start coding.
+
+Let's continue with the installation.
+
+## Installing VS code
+
+In previous posts, we demonstrated how to install programs using Pacman. Here,
+we want to use something else called _Snap_. Feel free to read this
+<a href="https://en.wikipedia.org/wiki/Snap_(software)"> Wikipedia</a> article
+to learn more about it. Why don't we use Pacman? Simply because Pacman doesn't
+provide VS code. If you want to get installation instructions, it’s often a good
+idea to visit the program’s homepage.
+
+First, we use Pacman to install Snap:
+
+    sudo pacman -S snapd
+
+Snap is a system service that runs in the background. We have to enable it:
+
+    sudo systemctl enable snapd
+    sudo systemctl start snapd
+
+Finally, we need to create a link:
+
+    sudo ln -s /var/lib/snapd/snap /snap
+
+Sounds all like black magic to you? Well, often it does for us as well :-D Let's
+keep things simple here: If you want to install something, you will find
+instructions with the help of Google.
+
+Before we can install VS code, we have to reboot the computer. This ensures that
+everything is set up correctly for Snap to work. Install VS code:
+
+    sudo snap install code --classic
+
+That's it!
+
+## Starting VS code for the first time
+
+You can run VS code either from the application menu in Manjaro or from the
+terminal:
+
+    code &
+
+Once VS code is started, you see a _Welcome_ screen, from where you directly can
+create files and start coding. But let's deal with some settings first. On the
+left bar, you can see several icons. If you hover over them with your mouse, you
+get some explanation.
+
+Let's start with the fifth icon from the top, which is labeled **Extensions**.
+Once you click on the icon, you can look for extensions on the marketplace. Good
+extensions can improve your coding experience a lot. So, it is wise to spend
+some time to find the extensions that meet your needs. But since it is hard to
+know your needs as a beginner, we prepared you a list of our favorite
+extensions. Before we show you how to install them, click on the gear wheel icon
+at the bottom and select **Settings**. Here, you can customize almost
+everything: Colors, cursor styles, auto-saving, and much more. When you have
+time, it’s worth checking out what is possible. But since we want to get coding
+fast, we share our settings with you.
+
+VS code supports so-called **Profiles**. Profiles store extensions and settings.
+It's possible to have multiple profiles to target different projects. For
+example, you might have one profile for Python and one for web programming using
```

Comment:
```suggestion
example, you might have one profile for Python and another one for web programming using
```
Also, don't think the "using HTML..." is really needed. Simply "web programming" should be enough.